servers/rendering/renderer_scene_occlusion_cull.cpp

```diff
@@ -124,6 +128,11 @@ void RendererSceneOcclusionCull::HZBuffer::resize(const Size2i &p_size) {
 }
 
 void RendererSceneOcclusionCull::HZBuffer::update_mips() {
+	// Keep this up to date as used for occlusion timers.
+	// Note this will wrap around with extremely long uptime and high FPS,
+	// so we must deal with wraparound in is_occluded().
+	occlusion_frame = Engine::get_singleton()->get_frames_drawn();
```

Comment:
`get_frames_drawn` returns a uint64_t, should we use the same for occlusion_frame so we can avoid the wraparound?

Reply:
The wraparound is dealt with in `renderer_scene_occlusion_cull.h`, line 196, it is fairly easy to detect and deal with:

```
// Detect wraparound.
// This can be considered somewhat optional,
// in case it is a bottleneck.
// The worst that will happen is that after wraparound
// an instance will not be occluded until it has been
// seen again (and thus the timeout is reset).
else if ((r_occlusion_timeout - occlusion_frame) >= 64) {
	r_occlusion_timeout = 0;
}
```

I think I did it as 32 bit originally to save on size in `InstanceData`, but maybe we won't have as many of these as e.g. 2D items, so maybe this is overkill, and it might be easier for readers.